src/app.js

```diff
@@ -1,21 +1,49 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const url = require('url');
+const fs = require('fs');
+
+const server = http.createServer((req, res) => {
+  const normalizedUrl = new url.URL(req.url, `http://${req.headers.host}/`);
+  const urlParts = normalizedUrl.pathname.slice(1).split('/');
+
+  if (urlParts[0] !== 'file') {
+    res.statusCode = 400;
+    res.statusMessage = 'Wrong path';
+
+    res.end('Request path must start with /file');
+
+    return;
+  }
+
+  let filePath = `./src/public/${urlParts.slice(1).join('/')}`;
+
+  if (filePath === './src/public/') {
+    filePath = './src/public/index.html';
+  }
+
+  if (filePath.split('.')[filePath.split('.').length - 1] === 'html') {
+    res.setHeader('Content-Type', 'text/html');
+  } else {
+    res.setHeader('Content-Type', 'text/plain');
```

Comment:
Use `application/octet-stream` for arbitrary files. E.g sending a `.jpg` with `text/plain` is wrong
```suggestion
    res.setHeader('Content-Type', 'application/octet-stream');
```